web/wp-content/themes/dctx2/template-rapidpass.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-p1ajsj.php, but found phpcs-scan-p1Ajsj.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).